api/keto/acl/admin/v1/admin_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package keto.acl.admin.v1;
+
+import "google/api/client.proto";
+
+option go_package = "github.com/ory/keto/api/keto/acl/admin/v1;admin";
+option csharp_namespace = "Ory.Keto.Acl.Admin.V1";
+option java_multiple_files = true;
+option java_outer_classname = "AdminServiceProto";
+option java_package = "sh.ory.keto.acl.admin.v1";
+option php_namespace = "Ory\\Keto\\Acl\\Admin\\V1";
```

Comment:
Yes, everything does! :)
As a [sample reference](https://github.com/googleapis/googleapis/blob/3ac5ef0436d8dfeb2ca0091dc7fa8012da1c85af/google/container/v1/cluster_service.proto#L25).

In case, is there a specific `java_package` domain ORY has?

Reply:
Yes, you nailed it already :wink: 
https://github.com/ory/sdk/tree/master/clients/hydra/java#getting-started